code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -1821,3 +1821,9 @@
 
 	icon_override = 'icons/vore/custom_clothes_vr.dmi'
 	item_state = "blindshades_mob"
+
+//Storesund97 - Aurora
+/obj/item/clothing/accessory/solgov/department/security
```

Comment:
this needs an /aurora at the end, otherwise it will overwrite the standard item.